.circleci/config.yml

```diff
@@ -0,0 +1,162 @@
+      - save_cache: *save_yarn_cache
+  build_artifacts:
+    description: 'Building artefacts'
+    steps:
+      - run: yarn build
+  lint:
+    description: 'Running linters'
+    steps:
+      - run: yarn run jest --projects jest.eslint.config.js jest.stylelint.config.js --maxWorkers=3 --reporters jest-silent-reporter
+  unit_test:
+    description: 'Running tests'
+    steps:
+      - run: yarn run jest --projects jest.test.config.js --maxWorkers=3 --reporters jest-silent-reporter
+  vrt_test:
+    description: 'Running Visual Regression Tests'
+    steps:
+      - run: yarn run-if:percy
+  build_playground:
+    description: 'Building playground'
+    steps:
+      - run: yarn playground:build
+  build_starter:
+    description: 'Building starter'
+    steps:
+      - run: yarn template-starter:build
+  publish_to_canary:
+    steps:
+      - install_dependencies
+      - build_artifacts
+      - run:
+          name: Publishing to npm registry
+          command: ./scripts/release_canary.sh
+
+jobs:
+  lint:
+    executor: node_10
+    steps:
+      - install_dependencies
+      - lint
+  unit_test:
+    executor: node_10
+    steps:
+      - install_dependencies
+      - unit_test
+  build:
+    executor: node_10
+    steps:
+      - install_dependencies
+      - build_artifacts
+  vrt_test:
+    executor: node_10
+    steps:
+      - install_dependencies
+      - build_artifacts
+      - vrt_test
+  publish:
+    executor: node_10
+    steps:
+      - install_dependencies
+      - build_artifacts
+      - publish_to_canary
+
+workflows:
+  version: 2
+
+  build_and_test:
+    jobs:
+      - lint:
+          filters:
+            branches:
+              only:
+                - refactor/use-circleci
+      - unit_test
+      - build:
+          requires:
+            - unit_test
+            - lint
+      - cypress/install:
+          name: install/e2e_playground
+          requires:
+            - build
+          yarn: true
+          # Building again is needed as Cypress writes/attaches to `~/`
+          # which is not configurable and we can not use as a root.
+          build: yarn build && yarn playground:build
+      - cypress/run:
+          name: run/e2e_playground
+          requires:
+            - install/e2e_playground
+          spec: 'cypress/integration/playground/**/*.js'
+          yarn: true
+          start: yarn playground:start:prod:local
+          wait-on: 'http://localhost:3001'
+          store_artifacts: true
+          # Parallel is needed as otherwise Cypress will not attach the workspace
+          parallel: true
+          parallelism: 1
+      - cypress/install:
+          name: install/e2e_starter
+          requires:
+            - build
+          yarn: true
+          # Building again is needed as Cypress writes/attaches to `~/`
+          # which is not configurable and we can not use as a root.
+          build: yarn build && yarn template-starter:build
+      - vrt_test:
```

Comment:
What's the reason for running the vrt tests after the e2e tests? 🤔 
There is no dependency between them, we can run them in parallel

Reply:
They run after the e2e tests finished installing. I ran them with the install before but it feels reasonable to have the actual tests: e2es and vrts run in parallel after the e2e are finished installing and running too. It's minor though. I would like to keep it like this it makes the workflow graph make a bit more sense.